cle/backends/__init__.py

```diff
@@ -497,10 +528,14 @@ def set_arch(self, arch):
         self.memory = Clemory(arch) # Private virtual address space, without relocations
 
     @property
+    def image_base_delta(self):
+        return - self.linked_base + self.mapped_base
```

Comment:
I had to blink a couple of times before I realized this wasn't a syntax error. Can you remove the space between the - and self.linked_base?